packages/react/src/components/Rating/Rating.base.tsx

```diff
@@ -10,7 +10,10 @@ const getClassNames = classNamesFunction<IRatingStyleProps, IRatingStyles>();
 const RatingStar = (props: IRatingStarProps) => {
   return (
     <div className={props.classNames.ratingStar}>
-      <Icon className={props.classNames.ratingStarBack} iconName={props.icon} />
+      <Icon
+        className={props.classNames.ratingStarBack}
+        iconName={props.fillPercentage === 50 || props.fillPercentage === 100 ? props.icon : props.unselectedIcon}
```

Comment:
It's counterintuitive, but the check here really does need to be for 0 not 50.
```suggestion
        iconName={props.fillPercentage === 0 || props.fillPercentage === 100 ? props.icon : props.unselectedIcon}
```

Reply:
Yeah! Sorry that was a blunder. yeah it should be 0 not 50